hydra-core/app/controllers/concerns/hydra/controller/download_behavior.rb

```diff
@@ -136,6 +135,9 @@ def send_file_contents
       def prepare_file_headers
         send_file_headers! content_options
         response.headers['Content-Type'] = file.mime_type
+        response.headers['Content-Length'] ||= file.size
+        # Prevent Rack::ETag from calculating a digest over body
+        response.headers['Cache-Control'] = 'no-cache'
```

Comment:
Can we set Last-Modified instead? It seems like setting no-cache could be detrimental especially for frequently downloaded assets like thumbnails. See. https://github.com/rack/rack/blob/master/lib/rack/etag.rb#L58